frontend/web/components/segments/SegmentRow/SegmentRow.tsx

```diff
@@ -107,13 +114,7 @@ export const SegmentRow: FC<SegmentRowProps> = ({
         className='table-column px-3'
         onClick={
           manageSegmentsPermission
-            ? () =>
-                history.push(
-                  `${document.location.pathname}?${Utils.toParam({
-                    ...Utils.fromParam(),
-                    id,
-                  })}`,
-                )
+            ? () => history.push(`${document.location.pathname}/${id}`)
```

Comment:
Ok interesting, the trailing slash needs to be cleaned up when there is a query param (`?featureSpecific` otherwise the next redirections fails (double slash):
`http://localhost:8080/project/25594/segments//493155`

To reproduce:
1. Go to a single segment page
2. Come back using the breadcrumb
3. Click any segment